src/modules/logicModule/LM_ManualExternalPriceSetter_v1.sol

```diff
@@ -126,17 +126,9 @@ contract LM_ManualExternalPriceSetter_v1 is
         uint issuancePrice_,
         uint redemptionPrice_
     ) external onlyModuleRole(PRICE_SETTER_ROLE) {
-        if (issuancePrice_ == 0 || redemptionPrice_ == 0) {
```

Comment:
Cleaned up this function from what was incoming/merged from the feature branch. Fyi